components/profile/ProfileDetail.vue

```diff
@@ -116,6 +118,8 @@
           :key="tab"
           :active="activeTab === tab"
           :text="tab"
+          :count="counts[tab]"
+          :show-active-check="false"
```

Comment:
why false?
on canary there is a checkmark on the active tab

Reply:
as per issue requirements 

![CleanShot 2023-10-01 at 14 04 31@2x](https://github.com/kodadot/nft-gallery/assets/44554284/0d883c20-ca7d-4399-be6b-5d93a7d03d3a)